### DISCUSSION TITLE
Improve FPS & Speed on Local Area Network.

### DISCUSSION THREAD

**Discussion by wilon** [rustdesk/rustdesk#2972] [General]
I build self server on LAN, I tried change parameters many times, the FPS(<10) and the Speed(<1M) always unsatisfactory.
Hope to improve FPS > 20 and the Speed > 5M.

**Comment by lihanchen** [comment]
I got solid 30 fps and 10m connection over wifi5 network. It might be your network. Maybe do local speed test with iperf3
